src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Models/DependencyModel.cs

```diff
@@ -42,6 +42,12 @@ internal class DependencyModel : IDependencyModel
             {
                 Flags = Flags.Except(DependencyTreeFlags.SupportsRemove);
             }
+
+            if (Properties.TryGetValue("Visible", out string visibleMetadata)
```

Comment:
@srivatsn I've fixed up a handful of places that were effectively forcing some kinds of graph nodes to be visible. With the SDK changes, we now hide all the façade assemblies under the "NETStandard.Library" node but still show netstandard.dll.